### PR TITLE
Docs cleanup

### PR DESCRIPTION
**Pull request by roberthou1**
Took a first pass at cleaning up the docs to only include info on how to use features!

I do need some help in pointing out where I can include Jetbrains tabs and specific info tho, please share feedback w/ me when you get the chance! :)

### CODE REVIEW

File: vscode/contextawareness/overview.mdx [hunk 1]
```diff
@@ -1,23 +1,54 @@
...
+For more information, please see our [@ Mentions](/vscode/chat/atmentions.mdx) page.
+
+<video
+  autoPlay
+  muted
+  loop
+  playsInline
+  className="w-full aspect-video"
+  src="https://exafunction.github.io/public/videos/chat/inline-mention.mp4"
+></video>
```
Comment (khou22) [inline vscode/contextawareness/overview.mdx:54]: Newline at end of file

File: vscode/contextawareness/overview.mdx [hunk 2]
```diff
@@ -1,23 +1,54 @@
 ---
 title: Overview
-description: 'What is Context?'
+description: ''
```
Comment (khou22) [inline vscode/contextawareness/overview.mdx:3]: Should every page have a description? I imagine it's used for SEO

**Review by khou22 — APPROVED** [review]
My general opinion is that things should be more visible by putting them as sections inside of the accordians. So there's more items besides just "Overview". Some notes:

* Let's add "Custom Instructions" to "Context Awareness" section
* Touch base with Xinyi to see if autocomplete should also include autocomplete power. Seems like it's in beta and we can include
* Include something inline fim to demonstrate that we can do this (https://codeium.com/blog/why-code-completion-needs-fill-in-the-middle)
* Autocomplete snoozing should be under "Autocomplete"
* Code lenses should talk about how to disable them. Would argue code lenses can be their own page since its settings, what "Explain" does. Talk about "Refactor"
* Slash commands under Chat as a new section?
* Section in the "Overview" of Chat that talks about "Copy", "Insert" etc.
* Mention stats for nerds, giving feedback, regenerate with context for chat
* Export conversation in chat
* How to view many conversations
* I would expect people to not watch the Chat video so lean into more text content
* Section on how to get help by using the "Debug Diagnostics"
* Chat settings should be included somewhere: theme preference, font size
* Include what you can do with your profile. Stats, public profile, usernames, referring others. This can be a top level grouping by itself.
* How to pop chat out into an editor so it can be dragged around as if it's an editor panel
* Tips for Autocomplete like instructing it via an inline comment, loading indicator in the status bar to tell when it's thinking, how to accept one word at a time, inline fim, where it can be very useful such as boilerplate and creating new react components
* "Models" probably deserves its own section in Chat so we can highlight this as a strength

<sub>_[Reviewed with Forge](https://codeium.com/forge/Exafunction/codeium-docs/4/review)_</sub>

**Comment by roberthou1** [comment]
> My general opinion is that things should be more visible by putting them as sections inside of the accordians. So there's more items besides just "Overview". Some notes:
> 
> * Let's add "Custom Instructions" to "Context Awareness" section
_done_
> * Touch base with Xinyi to see if autocomplete should also include autocomplete power. Seems like it's in beta and we can include
_will do_
> * Include something inline fim to demonstrate that we can do this (https://codeium.com/blog/why-code-completion-needs-fill-in-the-middle)
_there is a bullet point in Autocomplete/overview. figure this is enough for doc purposes?_
> * Autocomplete snoozing should be under "Autocomplete"
_changed "Toggle" to "Snoozing" in autocomplete/overview_
> * Code lenses should talk about how to disable them. Would argue code lenses can be their own page since its settings, what "Explain" does. Talk about "Refactor"
_I agree, I like it as a new page_
> * Slash commands under Chat as a new section?
_will add to chat/overview_
> * Section in the "Overview" of Chat that talks about "Copy", "Insert" etc.
_done_
> * Mention stats for nerds, giving feedback, regenerate with context for chat
_done_
> * Export conversation in chat
_done_
> * How to view many conversations
> * I would expect people to not watch the Chat video so lean into more text content
> * Section on how to get help by using the "Debug Diagnostics"
> * Chat settings should be included somewhere: theme preference, font size
_done_
> * Include what you can do with your profile. Stats, public profile, usernames, referring others. This can be a top level grouping by itself.
_will create profiles page under /gettingstarted_
> * How to pop chat out into an editor so it can be dragged around as if it's an editor panel
_will do_
> * Tips for Autocomplete like instructing it via an inline comment, loading indicator in the status bar to tell when it's thinking, how to accept one word at a time, inline fim, where it can be very useful such as boilerplate and creating new react components
_done_
> * "Models" probably deserves its own section in Chat so we can highlight this as a strength
_will do_
> 
> _[Reviewed with Forge](https://codeium.com/forge/Exafunction/codeium-docs/4/review)_

**Pull request merged**